esp/templates/qsd/bulk_move.html

```diff
@@ -11,7 +11,9 @@
 
 <h1>Move a block of static pages</h1>
 
-<p>The URL of a static page comes from two parts: the 'path' and the 'name.'  The path is the location in the ESP data tree where the page is anchored.  The name is the 'filename' of the page.  For example, the following parameters...</p>
+<p>The URL of a static page comes from two parts: the 'path' and the 'name.'
+The path is the location in the {{ settings.ORGANIZATION_SHORT_NAME }} data tree where the page is anchored.
+The name is the 'filename' of the page.  For example, the following parameters...</p>
 <ul>
 <li><b>Path:</b> Q/Web/about
```

Comment:
Wow, I thought we had gotten rid of all the Qs!  This QSD page would just have a URL /about/oobleck.html now, which probably needs no explanation other than maybe a "without the domain name".

Reply:
Made some changes. Does that look better?

Reply:
I think "string" is a bit jargon-y, just say "doesn't include the domain name" or something, but otherwise looks great, thanks!